packages/blocks/src/api/registration.js

```diff
@@ -277,9 +277,10 @@ export function registerBlockType( name, settings ) {
 /**
  * Registers a new block collection to group blocks in the same namespace in the inserter.
  *
- * @param {string} namespace The namespace to group blocks by in the inserter; corresponds to the block namespace
- * @param {Object} settings  An object composed of a title to show in the inserter, and an icon to show in the inserter
- *
+ * @param {string} namespace The namespace to group blocks by in the inserter; corresponds to the block namespace.
+ * @param {Object} settings  The block collection settings.
+ * @param {string} settings.title The title to display in the block inserter.
+ * @param {Object} settings.icon  The icon to display in the block inserter.
```

Comment:
Not sure how strict we want to be with these but the descriptions are not aligned.